### PR TITLE
Added a script to delete document from DP&U.

### PR DESCRIPTION
**Pull request by dharmez**
Added a bash script to delete a document from DP&U. Updated README.md.

### CODE REVIEW

File: sample-deployments/composer-orchestrated-process/DEPLOYMENT.md [hunk 1]
```diff
@@ -1,68 +1,4 @@
-# Deployment Guide
+# [Deployment Guide](../../README.md#deployment-guide)
```
Comment (eyalbenivri) [inline sample-deployments/composer-orchestrated-process/DEPLOYMENT.md:1]: Is there a `#deployment-guide` in the `../../README.md` file? I can't see it

**Review by eyalbenivri — COMMENTED** [review]
Left one comment about an anchor I think might be missing from the README.md file.

File: sample-deployments/composer-orchestrated-process/DEPLOYMENT.md (hunk 1 above)
Comment (dharmez) [reply sample-deployments/composer-orchestrated-process/DEPLOYMENT.md:1]: Yes, https://github.com/GoogleCloudPlatform/document-processing-and-understanding/blob/delete-doc/README.md#deployment-guide

**Review by dharmez — COMMENTED** [review]
Tested delete doc and restructured some documentation.

**Review by anuradha-bajpai-google — APPROVED** [review]
LGTM

**Comment by dharmez** [comment]
* Added a script to delete document from DP&U.

* Removed redundant documentation so that it does not get out of sync.

* Removed some more redundant documentation.

* Update READEME.md.

**Pull request merged**